readthedocs/projects/tasks/builds.py

```diff
@@ -146,7 +146,9 @@ def on_failure(self, exc, task_id, args, kwargs, einfo):
             )
         else:
             # Catch unhandled errors when syncing
-            log.exception('An unhandled exception was raised during VCS syncing.')
+            log.exception(
+                "An unhandled exception was raised during VCS syncing.", exc_info=exc
+            )
```

Comment:
```suggestion
            # Note we are using `log.error(exc_info=...)` instead of `log.exception` 
            # because this is not executed inside a try/except block.
            log.error(
                "An unhandled exception was raised during VCS syncing.", exc_info=exc
            )
```